docs/en/advanced_tutorials/initialize.md

```diff
@@ -1,3 +1,325 @@
+| [UniformInit](../api.html#mmengine.model.UniformInit)           |     Uniform     | Initialize the weight by union distribution, and initialize the bias with a constant，commonly used for convolution                      |
+| [KaimingInit](../api.html#mmengine.model.KaimingInit)           |     Kaiming     | Initialize the weight by `Kaiming` initialization, and initialize the bias with a constant. Commonly used for convolution                |
+| [Caffe2XavierInit](../api.html#mmengine.model.Caffe2XavierInit) |  Caffe2Xavier   | `Xavier` initialization in Caffe2, and `Kaiming` initialization in PyTorh with `fan_in` and `normal` mode. Commonly used for convolution |
+| [PretrainedInit](../api.html#mmengine.model.PretrainedInit)     |   Pretrained    | Initialize the model with the pretrained model                                                                                           |
+
+### Initialize the model with pretrained model
+
+Defining the `ToyNet` as below:
+
+```python
+import torch
+import torch.nn as nn
+
+from mmengine.model import BaseModule
+
+
+class ToyNet(BaseModule):
+
+    def __init__(self, init_cfg=None):
+        super().__init__(init_cfg)
+        self.conv1 = nn.Linear(1, 1)
+
+
+# Save the checkpoint.
+toy_net = ToyNet()
+torch.save(toy_net.state_dict(), './pretrained.pth')
+pretrained = './pretrained.pth'
+
+toy_net = ToyNet(init_cfg=dict(type='Pretrained', checkpoint=pretrained))
+```
+
+and then we can configure the `init_cfg` to make to load the pretrained model by calling `initi_weights()` after its construction.
+
+```python
+# Initialize the model with the saved checkpoint.
+toy_net.init_weights()
+```
+
+```
+08/19 16:50:24 - mmengine - INFO - load model from: ./pretrained.pth
+08/19 16:50:24 - mmengine - INFO - local loads checkpoint from path: ./pretrained.pth
+```
+
+If `init_cfg` is a `dict`, `type` means a kind of initializer registered in `WEIGHT_INITIALIZERS`. The `Pretrained` means `PretrainedInit`, which could help us to load the target checkpoint.
+All initializers have the same mapping relationship like `Pretrained` -> `PretrainedInit`, which strips the suffix `Init` of the class name. The `checkpoint` argument of `PretrainedInit` means the path of the checkpoint. It could be a local path or a URL.
+
+### Commonly used initialization methods
+
+Similarly, we could use the `Kaiming` initialization just like `Pretrained` initializer. For example, we could make `init_cfg=dict(type='Kaiming', layer='Conv2d')` to initialize all `Conv2d` module with `Kaiming` initialization.
+
+Sometimes we need to initialize the model with different initialization methods for different modules. For example, we could initialize the `Conv2d` module with `Kaiming` initialization and initialize the `Linear` module with `Xavier` initialization. We could make `init_cfg=dict(type='Kaiming', layer='Conv2d', override=dict(type='Xavier', layer='Linear'))`:
```

Comment:
Is it too early for `override` to occur here?

Reply:
Stupid me use copilot stupidly 😢 .